tests/integ/test_session.py

```diff
@@ -288,16 +292,21 @@ def test_list_files_in_stage(session, resources_path, local_testing_mode):
 
 @pytest.mark.localtest
 @pytest.mark.skipif(IS_IN_STORED_PROC, reason="Cannot create session in SP")
-def test_create_session_from_parameters(db_parameters, sql_simplifier_enabled):
+def test_create_session_from_parameters(
+    db_parameters, sql_simplifier_enabled, local_testing_mode
+):
     session_builder = Session.builder.configs(db_parameters)
     new_session = session_builder.create()
     new_session.sql_simplifier_enabled = sql_simplifier_enabled
     try:
         df = new_session.createDataFrame([[1, 2]], schema=["a", "b"])
         Utils.check_answer(df, [Row(1, 2)])
         assert session_builder._options.get("password") is None
-        assert new_session._conn._lower_case_parameters.get("password") is None
-        assert new_session._conn._conn._password is None
+        if not local_testing_mode:
+            assert new_session._conn._lower_case_parameters.get("password") is None
+            assert new_session._conn._conn._password is None
+        else:
+            assert isinstance(new_session._conn._conn._password, Mock)
```

Comment:
hopefully we don't need this if-else after completely mocking connector connection, I have added this test into JIRA SNOW-1368486